YapDatabase/YapDatabaseOptions.m

```diff
@@ -56,6 +58,7 @@ - (id)copyWithZone:(NSZone __unused *)zone
     copy->cipherPageSize = cipherPageSize;
 #endif
 	copy->aggressiveWALTruncationSize = aggressiveWALTruncationSize;
+    copy->enableMultiProcessSupport = enableMultiProcessSupport;
```

Comment:
OK indeed. Xcode convention is to use spaces and Yap convention is to use tabs ?